src/ui-kit/components/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,29 @@
+import { memo } from 'react';
+import { AvatarText, AvatarWrapper } from './Avatar.styled';
+
+function Avatar({
+  AvatarComponent = 'AB',
```

Comment:
The name of the props cannot be capitalized. And what's the point of this props if it's just children